packages/expo-modules-core/ios/Core/AppContext.swift

```diff
@@ -85,6 +85,23 @@ public final class AppContext: NSObject {
     }
   }
 
+  /**
+   The application identifier that is used to distinguish between different `RCTHost`.
+   It might be equal to `0`, meaning we couldn't obtain the Id for the current app.
+   It shouldn't be used on the Paper.
```

Comment:
```suggestion
   It might be equal to `0`, meaning we couldn't obtain the ID for the current app.
   It shouldn't be used on the old architecture.
```